packages/neo4j-driver/src/index.js

```diff
@@ -342,7 +343,7 @@ async function hasReachableServer (url, config) {
   }
 }
 
-const USER_AGENT = 'neo4j-javascript/' + VERSION
+const BOLT_AGENT = internal.boltAgent.fromVersion('neo4j-javascript/' + VERSION)
```

Comment:
Maybe move the definition to done before the usage (i know this was already this way, but I think it worth the small refactory).